lib/Cleantalk/ApbctWP/CleantalkRealPerson.php

```diff
@@ -51,23 +51,44 @@ public function getCommentAuthor($comment_author, $comment_id, $comment)
         $trp_popup_header = __('The Real Person!', 'cleantalk-spam-protect');
         $trp_author = $comment_author;
         $trp_author_bold = '<b>' . $comment_author . '</b>';
-        $trp_popup_text = __('Author %s acts as a real person and passed all tests against spambots. Anti-Spam by CleanTalk.', 'cleantalk-spam-protect');
-        $trp_popup_text = sprintf($trp_popup_text, $trp_author_bold);
+        $trp_popup_text_person = __('Author %s acts as a real person and verified as not a bot.', 'cleantalk-spam-protect');
+        $trp_popup_text_person = sprintf($trp_popup_text_person, $trp_author_bold);
+        $trp_popup_text_shield = __('Passed all tests against spam bots. Anti-Spam by CleanTalk.', 'cleantalk-spam-protect');
+
         $trp_comment_id = 'apbct_trp_comment_id_' . $comment_id;
-        $trp_title_popup_open_script = "
-            let popup = document.getElementById('$trp_comment_id');
-            popup.style.display = 'inline-flex';
-        ";
+        $trp_title_popup_open_script = "apbctRealUserBadgeViewPopup('$trp_comment_id');";
+        $trp_link_img_person = '/wp-content/plugins/cleantalk-spam-protect/css/images/real_user.svg';
```

Comment:
Use `SPBC_PATH . '/images'` instead of long hard-coded path

Reply:
I'll fix it, just for this APBCT_URL_PATH